src/components/form/contact.tsx

```diff
@@ -70,7 +74,7 @@ export const ContactForm: React.FC = () => {
         <GridItem xs={12} md={6}>
           <InputField
             required={true}
-            inputRef={register({ required: 'Ihr Name fehlt' })}
+            inputRef={register('name', { required: 'Ihr Name fehlt' })}
```

Comment:
Did that work before because it's implicitly given by the name attribute ?

Reply:
The register() broke because of the update from "6.15.8" to "7.8.7" (zum Vergleich: [https://react-hook-form.com/api/useform/register](https://react-hook-form.com/api/useform/register))